acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/PantheonNode.java

```diff
@@ -90,6 +90,8 @@
   private HttpRequestFactory httpRequestFactory;
   private Optional<EthNetworkConfig> ethNetworkConfig = Optional.empty();
   private boolean useWsForJsonRpc = false;
+  private boolean useAuthenticationTokenInHeaderForJsonRpc = false;
```

Comment:
Can probably simplify this boolean out by either making token an Optional<String> or checking token != null

Reply:
👍